packages/zipkin/test/trace.test.js

```diff
@@ -115,9 +115,7 @@ describe('Tracer', () => {
     ctxImpl.scoped(() => {
       const promise = trace.local('buy-smoothie', () =>
         Promise.delay(10)
-               .then(() => {
-                 throw new Error('no smoothies. try our cake');
-               })
+               .then(() => 'smoothie')
```

Comment:
Shouldn't we have a unit test for the error case as well?

Reply:
:) the red you are seein was me correcting code that did same check in 2 test cases! I fixed one to test success

Reply:
Can add another test to verify the error calback sets correct trace ids though..